workflow/artifacts/git/git.go

```diff
@@ -41,7 +56,11 @@ func (g *GitArtifactDriver) auth() (func(), transport.AuthMethod, []string, erro
 		if err != nil {
 			return nil, nil, nil, err
 		}
-		auth := &ssh2.PublicKeys{User: "git", Signer: signer}
+		var sshUser string
```

Comment:
Shouldn't the default `sshUser` be `git`?
```suggestion
		sshUser := "git"
```